src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -67,12 +67,25 @@
              {}
              amounts))
 
+(defn add-zero-values-to-network-values
+  [network-values all-possible-chain-ids]
+  (reduce
+   (fn [acc chain-id]
+     (let [route-value (get network-values chain-id)]
+       (assoc acc chain-id (or route-value (money/bignumber "0")))))
+   {}
+   all-possible-chain-ids))
+
 (defn token-available-networks-for-suggested-routes
-  [{:keys [balances-per-chain disabled-chain-ids]}]
+  [{:keys [balances-per-chain disabled-chain-ids only-with-balance?]}]
```

Comment:
I would say `only-with-balance` is not very clear. What is the purpose of this variable ? 🤔